practice/react-practice-one/src/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,99 @@
+/* Import dependencies */
+import PropTypes from 'prop-types';
+
+/* Import components */
+import Button from '../Buttons/Button';
+
+/* Import CSS */
+import './pagination.css';
+
+const Pagination = ({ currentPage, totalRecords, pageLimit, onPageChange }) => {
```

Comment:
@JennyDoJD should we need default value for props?

Reply:
I done